gazebo_ros_control/CMakeLists.txt

```diff
@@ -31,9 +33,11 @@ catkin_package(
     joint_limits_interface
     urdf
     angles
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME} default_robot_hw_sim
-  DEPENDS gazebo
```

Comment:
According to [catkin documentation](http://wiki.ros.org/catkin/CMakeLists.txt) that `DEPENDS` clause is fine to my eyes. My vote is for keeping it and debug where is exactly the error.